openpgp/packet/packet.go

```diff
@@ -455,6 +455,16 @@ func (cipher CipherFunction) KeySize() int {
 	return algorithm.CipherFunction(cipher).KeySize()
 }
 
+// IsSupported returns true if the cipher is supported from the library
+func (cipher CipherFunction) IsSupported() bool {
+	return algorithm.CipherFunction(cipher).KeySize() > 0
+}
+
+// IsAes returns true if the cipher is AES
+func (cipher CipherFunction) IsAes() bool {
+	return cipher.IsSupported() && cipher >= CipherAES128
```

Comment:
I think it's better to make this an explicit check that it's one of the three values, because in the standard there's also TwoFish and Camellia that have higher IDs than AES.